RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -60,6 +63,7 @@ void OnlineBeamSpotESProducer::fillDescriptions(edm::ConfigurationDescriptions&
   edm::ParameterSetDescription dsc;
   dsc.add<int>("timeThreshold", 48)->setComment("hours");
   dsc.add<double>("sigmaZThreshold", 2.)->setComment("cm");
+  dsc.add<double>("sigmaXYThreshold", 4.)->setComment("um");
```

Comment:
For my education: what are the guidelines/rules to follow in terms of units for configuration parameters? (is one free to choose whatever unit feels appropriate?). Here, the two parameters have very similar names, but very different units. I'm wondering if this can lead to confusion, or it is common practise.

Reply:
IMO expressing a cut on the beam transverse width (which is always in the ballpark of 10um in standard pp collisions and O(100um) for fat commissioning beams) in cm is more error prone than having mixed units as one has always to make mental conversions back and forth. 
To make it plain to pedestrians, I would suggest to name the parameters `sigmaXYThresholdInUm` and `sigmaZThresholdInCm` (though the latter implies the hassle of customizing the HLT menu in release), as well as adding comments in the `fillDescriptions` fields.

Reply:
> IMO expressing a cut on the beam transverse width (which is always in the ballpark of 10um in standard pp collisions and O(100um) for fat commissioning beams) in cm is more error prone than having mixed units as one has always to make mental conversions back and forth. 

I tend to agree

> To make it plain to pedestrians, I would suggest to name the parameters `sigmaXYThresholdInUm` and `sigmaZThresholdInCm` (though the latter implies the hassle of customizing the HLT menu in release), as well as adding comments in the `fillDescriptions` fields.

I don't think that such a renaming is needed. It is already written in the comment fields in the code, where people can look at in case of doubts.